pkgs/misc/vscode-extensions/default.nix

```diff
@@ -66,6 +90,18 @@ in
     extractNuGet = callPackage ./python/extract-nuget.nix { };
   };
 
+  skyapps.fish-vscode = buildVscodeMarketplaceExtension {
+    mktplcRef = {
+      name = "fish-vscode";
+      publisher = "skyapps";
+      version = "0.2.1";
+      sha256 = "0y1ivymn81ranmir25zk83kdjpjwcqpnc9r3jwfykjd9x0jib2hl";
+    };
+    meta = with stdenv.lib; {
+      license = stdenv.lib.licenses.mit;
```

Comment:
```suggestion
      license = licenses.mit;
```

Reply:
Thanks. I changed the license and deleted code from other PR.